### PR TITLE
Remove unused parts

### PR DESCRIPTION
**Pull request by joehoyle**

### CODE REVIEW

**Comment by szepeviktor** [comment]
@tcrsavage How about this one?

**Comment by tcrsavage** [comment]
@szepeviktor Are you asking about the merge conflict coming from https://github.com/humanmade/wordpress-pecl-memcached-object-cache/pull/10/files ?

Joe will presumably want to preserve that `&& ! $force` check when resolving

**Comment by mikeselander** [comment]
@joehoyle @tcrsavage this library is throwing an error in my local test suite and this PR would fix that issue. Is this still going to move forward and get included in HM platform?

File: object-cache.php [hunk 1]
```diff
@@ -278,13 +114,20 @@ function wp_cache_fetch_all() {
...
+function wp_cache_flush( $delay = 0 ) {
+	if ( version_compare( PHP_VERSION, '5.4.0', '>=' ) ) {
+		$caller = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS, 1 );
+		$caller = $caller[0];
+	} else {
+		$caller = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS );
+		$caller = $caller[0];
+	}
+
+	if ( 'cli' !== php_sapi_name() ) {
```
Comment (mikeselander) [inline object-cache.php:126]: Will this change allow `wp_cache_flush` to be called when running unit tests? WordPress core flushes the cache at the shutdown of every single test class of theirs which, when combined with this dropin, instantly kills a test suite.

Comment (joehoyle) [reply object-cache.php:126]: Hmm good catch. Maybe we should patch this on master. I don't really think this PR is likely to be merged right away, but seems this issue could be a good one to push through.

Comment (mikeselander) [reply object-cache.php:126]: Added #11 and optionally #12 to address test issues.

**Comment by szepeviktor** [comment]
How could I help to move this forward?

**Pull request closed without merging**